src/core/PWSprefs.cpp

```diff
@@ -135,7 +135,7 @@ const PWSprefs::boolPref PWSprefs::m_bool_prefs[NumBoolPrefs] = {
   {_T("EnableWindowTransparency"), false, ptApplication},   // application
   {_T("ShowMenuSeparator"), true, ptApplication},           // application
   {_T("AutoAdjColWidth"), false, ptApplication},            // application
-  {_T("ToolbarShowText"), false, ptApplication},            // application
+  {_T("ToolbarShowText"), true, ptApplication},             // application
```

Comment:
I prefer to see the labels on the toolbar buttons, but is there a technical reason to change the default behavior?